article/app/views/liveblog/liveBlogBodyContentAMP.scala.html

```diff
@@ -28,6 +29,11 @@
         @if(model.currentPage.currentPage.isArchivePage) {
             @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage, amp = true)
         }
+
+            <script type="application/ld+json">
```

Comment:
Talked to Jon and Diego and the consensus is to extract it all into common for now.